src/main/groovy/org/gradle/android/workarounds/RoomSchemaLocationWorkaround.groovy

```diff
@@ -199,6 +201,7 @@ class RoomSchemaLocationWorkaround implements Workaround {
      * of those tasks if only a single variant is being assembled.
      */
     static class RoomSchemaLocationMergeTask extends DefaultTask {
+        @Input
```

Comment:
I think this should be `@Internal`.  I explicitly did not want this task to be incremental (and I'm not sure `@Input` would work in an expected way here, either).